src/handler.ts

```
@@ -43,13 +68,222 @@ function setupAWS() {
+    } else {
+        return new AWS.S3({endpoint: 'http://s3.us-east-1.amazonaws.com', params: {Bucket: bucketName}, region: 'us-east-1'});
+    }
+}
+
+/**
+ * Copies files in s3 to location of new author
+ * @param token token for multipart file uploads
+ * @param fromCuid the cuid of the object to move
+ * @param oldAuthorAccessID the file access id of the old author
+ * @param newAuthorAccessID the file access id of the new author
+ */
+
+async function copyFiles(fromCuid, oldAuthorAccessID, newAuthorAccessID) {
+
+    const oldPrefix = `${oldAuthorAccessID.fileAccessId}/${fromCuid}`;
+    const newPrefix = `${newAuthorAccessID.fileAccessId}/${fromCuid}`;
+    try {
+        s3.listObjectsV2({Prefix: oldPrefix}, function (err, data) {
+            if (err) {
+                console.log(err, err.stack); // logs if an error occurs
+            } else {
+                console.log(data);
+            }
+            if (data.Contents.length) {
+                async.each(data.Contents, function(file, cb) {
+                    if (!file.Key.includes(`${fromCuid}.zip`) && !file.Key.includes(`.pdf`)) {
+                        console.log(file.Key);
+                        const params = {
+                            Bucket: bucketName,
+                            CopySource: bucketName + '/' + file.Key,
+                            Key: file.Key.replace(oldPrefix, newPrefix),
+                        };
+                        s3.copyObject(params, function(copyErr, copyData) {
+                            if (copyErr) {
+                                console.log (copyErr);
+                            } else {
+                                console.log (copyData);
+                            }
+                            cb();
+                        });
+                    }
+                });
+            }
+        });
+    } catch (err) {
+        console.log(err);
```

We should probably throw this error so if something happened we get a response back in the client.